python/src/geo.rs

```diff
@@ -37,7 +37,8 @@ pub(crate) struct PyGeoKeyDirectory {
     geog_azimuth_units: Option<u16>,
     #[pyo3(get)]
     geog_prime_meridian_long: Option<f64>,
-
+    #[pyo3(get)]
```

Comment:
Would you be able to add a docstring here noting that this isn't part of the spec but is proposed with the link in the PR summary

Reply:
for sure, added in [`0f4ba6a` (#131)](https://github.com/developmentseed/async-tiff/pull/131/commits/0f4ba6a88867aa4a3990cb873ca65ffe8e9a850b)